subsys/bluetooth/mesh/Kconfig

```diff
@@ -660,7 +660,15 @@ config BT_MESH_MODEL_EXTENSIONS
 	  Enable support for the model extension concept, allowing the Access
 	  layer to know about Mesh model relationships.
 
-if BT_SETTINGS
+menuconfig BT_MESH_CONTINUOUS_STORE
+	bool "Store mesh settings continuously"
```

Comment:
Same here. The current implementation isn't really about continuity but ensuring that when something changes that it gets automatically stored.